examples/production/docker-compose-prod.yml

```diff
@@ -166,6 +168,7 @@ volumes:
     driver: local
   essnapshotdata-prod:
     driver: local
+  # display test results from non-production volume:
```

Comment:
Beetje flauw, maar misschien is 'expose' hier duidelijker dan 'display'?
Je krijgt niet werkelijk iets te zien, maar je kunt er wel bij.

Reply:
3623477